hw/top_earlgrey/templates/toplevel.sv.tpl

```diff
@@ -47,7 +48,7 @@ for m in top['memory']:
 last_modidx_with_params = lib.idx_of_last_module_with_params(top)
 
 %>\
-module top_${top["name"]} #(
+module top_${top["name"]} import top_${top["name"]}_pkg::*; #(
```

Comment:
Why does this import need moving? I think you're using the name for e.g. the `outgoing_alert_*` signals with a complete path?

Reply:
I think that one can be removed. I though originally it was needed for the signal definitions.